tests/install/test.py

```diff
@@ -50,24 +56,29 @@ class TestNested:
             "821.1": "xcpng-8.2.1-2023",
             "83b2": "xcpng-8.3-beta2",
             "83rc1": "xcpng-8.3-rc1",
+            "83rc1-net": "xcpng-8.3-rc1-net",
             "ch821.1": "ch-8.2.1-23",
             "xs8": "xs8-2024-03",
         }[version],
         gen_unique_uuid=True,
         param_mapping={"version": "iso_version"})
-    @pytest.mark.answerfile(lambda firmware, local_sr: {
-        "base": "INSTALL",
-        # FIXME this overrides part of base data
-        "root": {"tag": "installation"} if local_sr == "nosr" else {
-            "tag": "installation", "sr-type": local_sr,
+    @pytest.mark.answerfile(
+        lambda firmware, local_sr, version: {
+            "base": "INSTALL",
+            # FIXME this overrides part of base data
+            "root": {"tag": "installation"} if local_sr == "nosr" else {
+                "tag": "installation", "sr-type": local_sr,
+            },
+            "source": {"type": "url", "text": NETINSTALL_URL[version]} if version in NETINSTALL_URL else {
```

Comment:
@ydirson this doesn't work because version (that is the iso_version) is **83rc1-net**. Is there a way to transform it using the marker *installer_iso* or is it just not good?